### DISCUSSION TITLE
How do you perform client request cancelation in v10?

### DISCUSSION THREAD

**Discussion by IgnusG** [trpc/trpc#2282] [Q&A] [answered]
I noticed that v10 abandoned the cancelable promise design returning instead a plain promise. How can you cancel a request that is already in progress?

**Comment by KATT** [comment, marked as answer]
Interesting, i think this is something we just have missed, I think we should do something here with AbortController!

**Reply by KATT** [reply]
See #2283 

Thanks for bringing this up!
